.travis.yml

```diff
@@ -12,6 +12,17 @@ before_script:
 git:
   depth: 99999
 
+if: tag IS blank
+
+deploy:
+  provider: releases
+  api_key:
+    secure: CHANGEME
```

Comment:
I suppose this would be a travis secret? Any link to something already doing this?

Reply:
https://docs.travis-ci.com/user/deployment/releases/#Authenticating-with-an-OAuth-token
https://docs.travis-ci.com/user/encryption-keys/